packages/react-instantsearch-dom/src/components/List.js

```diff
@@ -61,13 +61,7 @@ class List extends Component {
   };
 
   renderItem = (item, resetQuery) => {
-    const items = item.items && (
-      <ul className={this.props.cx('list', 'list--child')}>
-        {item.items
-          .slice(0, this.getLimit())
-          .map(child => this.renderItem(child, item))}
-      </ul>
-    );
+    const isItemHasChildren = item.items && Boolean(item.items.length);
```

Comment:
`hasChildren` should be clear enough too right? This name now is pretty unclear :)

Reply:
Indeed. But I prefer to be explicit with something like `itemHasChildren`. WDYT?

Reply:
itemHasChildren also would be fine :)